### ISSUE TITLE
Test

### ISSUE THREAD

**Issue by Belami02** [ftm-2005/triager-tester#2]
TriagerX there!

**Comment by ftm-2005** [comment]
TriagerX Hello… is this think working??

**Comment by Belami02** [comment]
TriagerX Hello… is this think working??

**Comment by ftm-2005** [comment]
> TriagerX there!

I don’t think this works for description of an issue cuz I don’t think it counts as a issue comment… if we change the on issue commented to issue it might work

**Comment by Belami02** [comment]
> > TriagerX there!
> 
> I don’t think this works for description of an issue cuz I don’t think it counts as a issue comment… if we change the on issue commented to issue it might work

I guess it should work whenever the string "TriagerX" is at the start of the a comment. Check mine below
<img width="1053" alt="Screenshot 2024-06-01 at 3 40 27 PM" src="https://github.com/ftm-2005/triager-tester/assets/125438939/108a5ce6-fc83-4411-b3b7-321414c83830">

**Comment by ftm-2005** [comment]
> > > TriagerX there!
> > 
> > 
> > I don’t think this works for description of an issue cuz I don’t think it counts as a issue comment… if we change the on issue commented to issue it might work
> 
> I guess it should work whenever the string "TriagerX" is at the start of the a comment. Check mine below <img alt="Screenshot 2024-06-01 at 3 40 27 PM" width="1053" src="https://private-user-images.githubusercontent.com/125438939/335801898-108a5ce6-fc83-4411-b3b7-321414c83830.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTcyNDcyNTYsIm5iZiI6MTcxNzI0Njk1NiwicGF0aCI6Ii8xMjU0Mzg5MzkvMzM1ODAxODk4LTEwOGE1Y2U2LWZjODMtNDQxMS1iM2I3LTMyMTQxNGM4MzgzMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjAxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYwMVQxMzAyMzZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kMzM4YmQzNzcyMWZlZTdiYzJhMDMwN2NlYTZmNGYyMDdmM2VhNmQ4NWZlYWZmYmJmMmQ0OWEzNmIwZGY1YjgwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.WWea7O-SCQte6orUWx2JbxVkc6F0wJQUJp_QyzrVbIg">

Oh I can’t see that so I thought it didn’t work

**Comment by ftm-2005** [comment]
TriagerX name
